tests/test_flow/test_reloading.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+import tempfile
+import sys
+import time
+
+
+@pytest.fixture(scope="function")
+def make_tmp_flow_module():
+    # We want to be able to do import statement on the temporary module, so we
+    # create a temporary directory here and add it to sys.path.
+    tmp_dir = tempfile.TemporaryDirectory()
+    sys.path.insert(0, tmp_dir.name)
+
+    def _make_tmp_flow_module(var_name, var_value):
+        with open(f"{tmp_dir.name}/tmp_flow_module.py", "w") as tmp_file:
```

Comment:
If you just want to write some data to a file, you can do it like this:

    (tmp_dir / "tmp_flow_module.py").write_text("""...""")

Reply:
Oh, I just noticed your `tmp_dir` is a `TemporaryDirectory` object, which might not support the `/` operator and so on. An easier way to get a temporary directory is to use the built-in `tmp_path` fixture, as in:

    def make_tmp_module(tmp_path):
        sys.path.insert(0, str(tmp_path))

        ...
        (tmp_path / "tmp_flow_module.py").write_text(...)
        ...

Reply:
Thanks!